File: problems/3005/Solution.cs
public class Solution {
    public int MaxFrequencyElements(int[] nums) {
        int[] freq = new int[101];
        int best = 0;
        foreach (int x in nums) best = System.Math.Max(best, ++freq[x]);
        int ans = 0;
        foreach (int count in freq) if (count == best) ans += count;
        return ans;
    }
}

File: problems/3005/solution.rs
impl Solution {
    pub fn max_frequency_elements(nums: Vec<i32>) -> i32 {
        let mut freq = [0i32; 101];
        let mut best = 0;
        for x in nums {
            freq[x as usize] += 1;
            best = best.max(freq[x as usize]);
        }
        freq.into_iter().filter(|&c| c == best).sum()
    }
}
